Filter.php

```diff
@@ -37,6 +37,8 @@ public function __construct($field, $value, $operator = '=')
      */
     public function addFilter($field, $value, $operator = '=')
     {
+        //Encode any + signs in the value so we don't lose them in the urlencode/decode process
+        $value = str_replace('+', '%2B', $value);
```

Comment:
How about url encoding all data/params that's being passed, so this doesn't happen with anything else? Maybe somewhere right before sending the request to the API.

Reply:
I tried that first and couldn't get it to work. Let me look again.